homeassistant/components/camera/netatmo.py

```diff
@@ -0,0 +1,98 @@
+"""
+Support for the NetAtmo Weather Service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.netatmo/
+"""
+import logging
+import requests
+
+from homeassistant.components.camera import Camera
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ["netatmo"]
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_HOME = 'home'
+ATTR_CAMERAS = 'cameras'
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup access to Netatmo Welcome cameras."""
+    netatmo = get_component('netatmo')
+    home = config.get(CONF_HOME, None)
+    data = WelcomeData(netatmo.NETATMO_AUTH, home)
```

Comment:
if NETATMO_AUTH is bad then the netatmo component fails to load and then home-assistant can't load the camera.netatmo  component (as their is dependency) 
error message: homeassistant.bootstrap: Unable to prepare setup for platform camera.netatmo because dependency netatmo could not be initialized